docs/conf.py

```diff
@@ -560,10 +560,10 @@ def linkcode_resolve(domain, info):
     os.mkdir(docgen_path)
 
 for desc in descriptions:
-    fname = os.path.join(docgen_path, 'projectq.{}.rst'.format(desc.name))
+    fname = os.path.join(docgen_path, f'projectq.{desc.name}.rst')
     lines = None
     if os.path.exists(fname):
-        with open(fname, 'r') as fd:
+        with open(fname) as fd:
```

Comment:
Why remove the `'r'`?

Reply:
According to the Python standard, `'r'` is the default value for the mode. And therefore removed by `pyupgrade`